calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,514 @@
+| Threshold breach symptoms        | {{prodname}} network policies on affected nodes could be out-of-sync. |
+| Threshold breach recommendations | Check calico-node logs, rollout restart calico-node if needed. |
+| Priority level                   | Recommended.                                                 |
+
+### Felix cluster policies
+
+| Felix cluster policies           |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *felix_cluster_num_policies*                                 |
+| Explanation                      | Total number of {{prodname}} network policies in the cluster. |
+| Threshold value recommendation   | Because {{prodname}} is a distributed system, the number of policies should be generally consistent across all nodes.  It is expected to have some skew between nodes for a short time while they sync, however, they should never be out of sync for very long. |
+| Threshold breach symptoms        | If nodes are out of sync for a long time, calico-nodes may be having issues or experiencing resource contention.  Check the Errors Plot to see if there are any iptables errors reported. |
+| Threshold breach recommendations | Redeploy calico-node if issues are seen, and increase resources if needed. |
+| Priority level                   | Optional.                                                    |
+
+
+### Felix active local policies
+
+| Felix active local policies      |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *felix_active_local_policies*                                |
+| Explanation                      | Total number of network policies deployed on a per-node basis. |
+| Threshold value recommendation   | ??                                                           |
+| Threshold breach symptoms        | ??                                                           |
+| Threshold breach recommendations | Redeploy calico-node if issues are seen, and increase resources if needed. |
+| Priority level                   | Recommended.                                                 |
+
+### Felix open FDS
+
+| Felix open FDS                   |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *sum by (pod) (process_open_fds{pod=~"calico-node.*"})*      |
+| Explanation                      | Total number of opened file descriptors per calico-node pod. |
+| Threshold value recommendation   | ??                                                           |
```

Comment:
The threshold is `process_max_fds` , you don't want to be anywhere near the maximum.

Reply:
Fixed